packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+
+### Keyboard
+
+#### Selectable cards
+
+![Visual interaction of selectable cards using the keyboard](./assets/selectable-interaction-keyboard.png)
+
+1. Rest Default state is ‘unselected’
+2. Focus Focus rectangle is shown on card
+3. Focus Use spacebar to toggle state (selected/unselected)
+
+### Non-interactive cards
+
+![Visual interaction of non-interactive cards using the keyboard](./assets/non-interactive-interaction-keyboard.png)
+
+Keyboard interaction
+
+1. Focus Initial focus is set on card, to access items within the card use EnterKey or Spacebar
+2. Focus-Inside Focus moves to first actionable item in the card. Use ArrowKeys or TabKey to move focus to child elements within the card. To exit card use EscKey, to move focus back to card [parent]
+3. Checkbox-Focus Depending on your tab order of items within the card, the number of keystrokes will vary. Once focus is set on the checkbox by default it will be set to unselected
+4. SpaceBar or EnterKey will toggle both Card and Checkbox to selected state. Keying again will toggle to unselected
+
+### Context menu
+
+![Visual interaction of the context menu on cards using the keyboard](./assets/context-interaction-keyboard.png)
+
+1. Rest
+2. OnFocus - Focus rect. is set on card. Use shiftKey and press F10 to invoke context menu
+
+   `aria-haspopup="menu"`
+
+   `aria-expanded="false"`
+
+3. ShowSet focus on context menu
+
+   `aria-expanded="true"`
+
+4. Hide/Dismiss - Use EscKey to hide/dismiss
+
+   `aria-expanded="false"`
+
+## Accessibility
+
+> ⚠️ Note: All other accessibility information, not covered in this section, is provided throughout the spec.
+
+Each card is of `role="group"`.
+
+### Narrator guidance
+
+#### Selectable cards
+
+![Visual interaction of keyboard usage with selectable cards](./assets/selectable-narrator.png)
+
+1 Focus + unselected
+
+1. Label: Select this card
+2. Role: Group
+3. State: Focus + unselected
+4. Helper text: Use enterKey or spacebar to toggle state to selected
+
+2 Focus + selected
+
+1. Label: Unselect this card
+2. Role: Group
+3. State: Focus + selected
+4. Helper text: Use enterKey or spacebar to toggle state to unselected
+
+#### Non-interactive cards
+
+![Visual interaction of keyboard usage with non-interactive cards](./assets/non-interactive-narrator.png)
+
+1 Focus + unselected
+
+1. Label: App Card
+2. Role: Group
+3. State: Focus
+4. Helper text: Use enterKey to access controls within the card
+
+2 Focus (inside)
+
+1. Label: Open App
+2. Role: Button
+3. State: Focus
+4. Helper text: Use enterkey to open app in new window
+
+3 Focus Checkbox unselected
+
+1. Label: Share App
+2. Role: Button
+3. State: Focus
+4. Helper text: Use enter key to open share dialog
+
+4 Focus Checkbox selected
+
+1. Label: Select card
+2. Role: Checkbox
+3. State: Unselected
+4. Helper text: Use spacebar or enter key to select this item
+
+#### Disabled
+
+![Two disabled cards. One is unselected and one is selected](./assets/disabled-narrator.png)
```

Comment:
Are we implicitly saying that we will allow focus on disabled Cards? I don't think this is something we should allow.

Reply:
The point here is to portray disabled cards in different states. A selected card **is not** a focused card.
I can see this behavior useful in scenarios where you have cards selected but you're in a read-only mode. Your card is selected, but disabled as you can't change it in read-only.

Reply:
My bad, I got confused by the borders around the Cards but I see now that they were only there to emphasize them, please disregard this comment.